clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -3319,11 +3319,20 @@ private KafkaConsumer<String, String> consumerForCheckingTimeoutException(GroupP
         ConsumerPartitionAssignor assignor = new RangeAssignor();
 
         final KafkaConsumer<String, String> consumer = newConsumer(groupProtocol, time, client, subscription, metadata, assignor, false, groupInstanceId);
```

Comment:
Also, we should create consumer AFTER responses are ready. Otherwise, we may miss the FIND_COORDINATOR request